api/v1alpha3/awscluster_types.go

```diff
@@ -52,6 +52,10 @@ type AWSLoadBalancerSpec struct {
 	// Scheme sets the scheme of the load balancer (defaults to Internet-facing)
 	// +optional
 	Scheme *ClassicELBScheme `json:"scheme,omitempty"`
+
+	// Subnets specifies the subnets that should be used by the load balancer
+	// +optional
+	Subnets Subnets `json:"subnets,omitempty"`
```

Comment:
Can you provide an example in the godoc section on how to use this? From the implementation it looks like the AvailabilityZone field is required for example, otherwise it can fail / error, is that correct?

Reply:
Yes, in my usage I did need to specify availabilityZone. Would it be overreach to remove that requirement as part of this PR?

Happy to update the godoc section, but I'm new to godoc. Is it just a matter of adding information to the comments?